src/Symfony/Bundle/DebugBundle/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+4.1.0
+-----
+
+ * Added `debug.server_dump` config option allowing to enable and configure the `ServerDumper` and `ServerDumpCommand`
```

Comment:
should also tell about the `server:dump` command, as it's really the bundle that provides it

Reply:
done